php/WP_CLI/Runner.php

```diff
@@ -1487,6 +1486,10 @@ function( $url, $path, $scheme, $blog_id ) {
 			4
 		);
 
+		// Set up hook for WP-CLI packages to use.
```

Comment:
This is misleading, WP-CLI packages don't need this.
It should be something like:
```
// Set up hook for plugins & themes to conditionally add WP-CLI commands.
```